### DISCUSSION TITLE
problem. can you help?

### DISCUSSION THREAD

**Discussion by jorgegarmilla** [Shpota/github-activity-generator#62] [Q&A] [answered]
Hi. receive this error. "ssh: connect to host outlook.com port 22: Connection timed out. fatal: Could not read from remote repository."
I am not sure If I set up correctly the parameters. 
If my repository is https://github.com/jorgegarmilla/test-graph-generator and my email is jorge.garmilla@outlook.com 
should be python contribute.py --repository=jorge.garmilla@outlook.com:jorgegarmilla/test-graph-generator.git

what I am doing wrong. 
thanks!

**Comment by Shpota** [comment, marked as answer]
Hi,

You added your email to the `--repository` argument, which is why it fails. It should have been `git@github.com:jorgegarmilla/test-graph-generator.git`. However, it will still not work because your repository is already initialized. 

I recommend learning the basics of Git before running this script and generally not running anything without understanding what you do.

**Comment by kahrvba** [comment]
hi sir, your project is amazing and it's worth trying it out, but now I want to undo this and get back to my normal repository commits, can you help me out ?
